smalldata_tools/ana_funcs/droplet2Photons.py

```diff
@@ -8,71 +8,256 @@
 from scipy import sparse
 import time
 from smalldata_tools.DetObject import DetObjectFunc
+from numba import jit
+from numba.typed import List as NTL
 
 class droplet2Photons(DetObjectFunc):
-    '''
-    return_img: whether or not to return the img with how many photons at each coordinate
-    threshold: # (noise sigma for) threshold
+    """
+    aduspphot: expected photon energy
+    offset: offset to convert number of photons into energy boundaries (defaults to 0.5*aduspphot)
+    photpts: energy boundaries for given number of photons (defaults to offset + N*aduspphot)
+    one_photon_info: energy range for a single photon ((defaults to [offset, offset+N*aduspphot])
```

Comment:
Duplicate argument with one_photon_limits

Reply:
argument was not duplicate, info line was